VirtoCommerce.Platform.Web/App_Start/FilterConfig.cs

```diff
@@ -10,10 +10,10 @@ public class FilterConfig
     {
         public static void RegisterGlobalFilters(GlobalFilterCollection filters)
         {
-            filters.Add(new HandleErrorAttribute());
-			filters.Add(new AiHandleErrorAttribute());
+            //filters.Add(new ErrorHandler.AiHandleErrorAttribute());
```

Comment:
Could you explain why you commented on this line?

Reply:
The line is exactly the same as above 			filters.Add(new AiHandleErrorAttribute());